quil-rs/src/expression/simplification.rs

```diff
@@ -268,7 +269,7 @@ fn expression_to_recexpr(expression: &Expression) -> RecExpr<Expr> {
 fn recexpr_to_expression(recexpr: RecExpr<Expr>) -> Result<Expression, SimplificationError> {
     fn helper(nodes: &[Expr], i: usize) -> Result<Expression, SimplificationError> {
         match nodes[i] {
-            Expr::Pi => Ok(Expression::PiConstant),
+            Expr::Pi => Ok(Expression::Number(PI.into())),
```

Comment:
👏

Reply:
that's the one, thank you for the pointers